### ISSUE TITLE
Whitelist remote IPs to avoid DDoS

### ISSUE THREAD

**Issue by troyswanson** [scorecards/listener#1]
Prevent the system from being spammed with bogus data by keeping a whitelist of remote IPs. Disregard packets from anyone else. Eventually this should move up the stack to a firewall that can null route non-whitelisted traffic, but for now this is fine.

**Comment by troyswanson** [comment]
Could be a simple matter of storing a list in the `whitelist` key in Redis.

LPUSH and LRANGE could provide pretty much all of this functionality

```
127.0.0.1:6379> LPUSH whitelist "192.168.1.1"
(integer) 1
127.0.0.1:6379> LPUSH whitelist "192.168.1.10"
(integer) 2
127.0.0.1:6379> LRANGE whitelist 0 -1
1) "192.168.1.10"
2) "192.168.1.1"
```

**Comment by vel0city** [comment]
Would be good for it to accept slash notation, allowing for whole networks. So you could do something like 192.168.0.0/24 or 12.34.56.7/32.

**Comment by troyswanson** [comment]
Good idea. Eventually it should also support IPv6 addresses. Let's start with the simple whitelist for now though to get the foundation down.
